app/views/admin/divisions/index.html.slim

```diff
@@ -6,6 +6,7 @@
 = render 'layouts/admin/tables'
 
 = render "divisions_grid"
+- @no_records = @divisions_grid.current_page_records.length < 1
```

Comment:
No records means that no records match a specific set of filters.

Wice grid does not have a built-in way to display to the user in a more UX-friendly fashion that no records match the filters used. So this variable helps with that.

Per the Wice grid documentation, the method current_page_records *must* be used after the divisions grid. It could be included later in the page, but cannot be included in the controller because `define_grid(@divisions_grid, ...)` is defined in the view.